subprojects/core/src/main/java/org/gradle/api/internal/tasks/execution/ResolveBuildCacheKeyExecuter.java

```diff
@@ -80,6 +135,7 @@ public TaskExecuterResult execute(TaskInternal task, TaskStateInternal state, Ta
     }
 
     private void resolve(final TaskInternal task, final TaskExecutionContext context) {
+        // TODO:lptr Review this comment
```

Comment:
The inputs are now snapshotted in `ResolveBeforeExecutionStateTaskExecuter`. I guess there needs to be a change in the scan plugin, since we capture snapshotting time now as well. The time captured by this build operation is only the time it takes to calculate the build cache key from the fingerprints.

Reply:
Yes. We need some structural changes in tracking build operations to accommodate this. I'm talking to @ldaley about how to proceed on this.

Reply:
The structural changes are in.